test/e2e/configure_cni_test.go

```diff
@@ -64,6 +65,23 @@ func testConfigureCNI(t *testing.T) {
 	isConfiguredCorrectly, err := isCNIConfigured(t, kubeletLogPath)
 	require.NoError(t, err, "Error reading kubelet log")
 	assert.True(t, isConfiguredCorrectly, "CNI was not configured correctly")
+
+	// Get fully qualified path for kubelet
+	path, err := getSvcPath(bootstrapper.KubeletServiceName)
+	require.NoError(t, err, "Could not get kubelet arguments")
+	// Kubelet arguments with paths that are set by bootstrapper
+	// Does not include node-labels and container-image since their paths do not depend on underlying OS
+	checkPathsFor := []string{"--bootstrap-kubeconfig", "--cloud-config", "--config", "--kubeconfig", "--log-file",
+		"--cert-dir", "--cni-bin-dir", "--cni-conf-dir"}
+	populateKubeletArgs(path)
+	for _, key := range checkPathsFor {
+		if value, ok := kubeletArgs[key]; ok {
+			assert.NotEqualf(t, 1, len(strings.Split(value, string(os.PathSeparator))),
+				"Path not correctly set for %s", key)
+		} else {
+			assert.Failf(t, "Key %s not found in kubelet arguments", key)
+		}
+	}
```

Comment:
This can be made into a common function across `configure_cni_test.go` and `bootstrapper_test.go`. The only things that change are the args you check the path for.